packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -623,8 +646,28 @@ export class StatsAnalyzer extends EventsScope {
               this.statsResults.resolutions['video-send'].send.framesSent
             );
           }
-        }
 
+          // Video is encoded but frames are not sent
+          if (
+            currentStats.framesEncoded !== previousStats.framesEncoded &&
+            (currentStats.framesSent === previousStats.framesSent || currentStats.framesSent === 0)
```

Comment:
in here we're reading framesSent from currentStats/previousStats, while the code in lines 640-648 is using `this.statsResults.resolutions['video-send'].send.framesSent` and `this.lastStatsResults.resolutions['video-send'].send.framesSent` - why the difference? are these always the same? this is confusing....

Reply:
These values are same. Changed to `this.statsResults.resolutions['video-send'].send.framesSent` and `this.lastStatsResults.resolutions['video-send'].send.framesSent` to keep it uniform.

Reply:
there are still many places where we use `currentStats`/`previousStats` and `this.statsResults.resolutions` / `this.lastStatsResults.resolutions`, are these values all the same for sure? can we use just `currentStats`/`previousStats` everywhere? I know some of this is existing code, but since you're changing it, you might as well try to improve it.

Reply:
I checked. These values are the same. Changed to current and previous stats